### ISSUE TITLE
Automatic builds of packages

### ISSUE THREAD

**Issue by hickey** [kn6plv/meshchat#14]
It took me quite a  bit to find that all the active development of MeshChat is now here rather than on K7FPV's version. I wish the ardenmesh.org site would call more attention to this new version rather than direct everyone towards K7FPV's site.

Anyway, one thing I think that could help quite a bit is to automate the creation of packages to install this version of MeshChat. It would also be beneficial to actually go through a proper release procedure so that versions get tagged properly. 

I would like to help you with this process so that there is a place that can be referenced where AREDN node operators can find installable versions of MeshChat.

**Comment by hickey** [comment]
I have rolled back through the commits and created tags for each release. At least as best as I can ascertain. I can not push the tags here unless I am added as a repo member. 

![Screen Shot 2023-06-10 at 10 13 37 AM](https://github.com/kn6plv/meshchat/assets/39609/f04b7b46-4bbf-4ae6-95a0-7e8d6e91285b)

**Comment by aanon4** [comment]
Morning Gerald. As I'm sure is obvious, I need help with this project as my other commitments to AREDN means it get's little love. I'll add you as a project member.

**Comment by hickey** [comment]
I can certainly understand that. I appreciate getting added to the project and I have pushed the tags to the project. 

I have also started to reorganize the files so that they are code focused instead of being package based like it is now. That way people will be able to find the components easier and the relationships will be more apparent. I have started it on a branch in my fork of the repo. Here it is if you want to take a look: https://github.com/hickey/meshchat/tree/reorg. There is still some more work to be done and then I need to start adding GitHub actions so that the packages can be built. 

Once I get a bit further along, I will create a work in progress PR so that comments can be captured as to if everything is laid out in a reasonable fashion.
